src/cbc_sdk/platform/users.py

```diff
@@ -262,6 +294,192 @@ def reset_google_authenticator_registration(self):
         self._cb.delete_object(url)
 
+    @classmethod
+    def bulk_create(cls, cb, user_templates, profile_templates):
+        """
+        Creates a series of new users.
+
+        Args:
+            cb (CBCloudAPI): A reference to the CBCloudAPI object.
+            user_templates (list): List of templates for users to be created.
+            profile_templates (list): List of profile templates to be applied to each user.
+
+        Returns:
+            list: List of User objects that were just created.
+        """
+        my_profiles = normalize_profile_list(profile_templates)
+        return_users = []
+        for template in user_templates:
+            my_templ = copy.deepcopy(template)
+            my_templ['org_id'] = 0
+            my_templ['role'] = 'DEPRECATED'
+            if 'auth_method' not in my_templ:
+                my_templ['auth_method'] = 'PASSWORD'
+            if my_profiles:
+                my_templ['profiles'] = my_profiles
+            return_users.append(User._create_user(cb, my_templ))
+        return return_users
+
+    @classmethod
+    def bulk_add_profiles(cls, users, profile_templates):
+        """
+        Add the specified profiles to the specified users' grants.
+
+        Args:
+            users (list): List of User objects specifying users to be modified.
+            profile_templates (list): List of profile templates to be added to the users.
+        """
+        my_profiles = normalize_profile_list(profile_templates)
+        for user in users:
+            user._internal_add_profiles(my_profiles)
+
+    @classmethod
+    def bulk_disable_profiles(cls, users, profile_templates):
+        """
+        Disable the specified profiles in the specified users' grants.
+
+        Args:
+            users (list): List of User objects specifying users to be modified.
+            profile_templates (list): List of profile templates to be disabled.
+        """
+        my_profiles = normalize_profile_list(profile_templates)
+        for user in users:
+            user._internal_disable_profiles(my_profiles)
+
+    @classmethod
+    def bulk_disable_all_access(cls, users):
+        """
+        Disables all access profiles held by the listed users.
+
+        Args:
+            users (list): List of User objects specifying users to be disabled.
+        """
+        for user in users:
+            user.disable_all_access()
+
+    @classmethod
+    def bulk_delete(cls, users):
+        """
+        Deletes all the listed users.
+
+        Args:
+            users (list): List of User objects specifying users to be deleted.
+        """
+        for user in users:
+            user.delete()
+
+    def disable_all_access(self):
+        """Disables all access profiles held by ths user."""
+        grant = self.grant()
+        for profile in grant.profiles_:
+            profile.disabled = True
+        grant.save()
+
+    def change_role(self, role_urn, org=None):
```

Comment:
It can be either. We pass it through `normalize_org()` to turn it to a URN internally.

Reply:
hmm. should this be broken into two properties then? `org_key` and `org_urn` as `org` is not specific and we should refer to the specific property.

Reply:
This does match the way that we use org keys/URNs on things like `GrantBuilder` and `ProfileBuilder`.

Reply:
At a minimum we need a better doc string to describe what should be specified as the current doc string does not indicate that either could be specified. 

I'm also noticing that we implemented `def normalize_org()` in both grants and user python files. We should probably import it from grants.py instead of recreating it here

Reply:
I've updated the docstring.  And I'm actually having `grants.py` import `normalize_org()` from `users.py`, since both the "normalize" functions are in the latter. Easier consolidation.